querybook/server/lib/query_review/base_query_review_handler.py

```diff
@@ -62,27 +82,97 @@ def initiate_query_peer_review_workflow(
             reviewer_ids=reviewer_ids,
             session=session,
         )
+
         notify_reviewers_of_new_request(
             query_review_id=query_review.id,
             uid=uid,
+            template_name=self.notification_template,
+            additional_params=self.notification_template_params,
             session=session,
         )
+
         return query_review.to_dict()
 
-    def approve_review(self, query_review_id: int, reviewer_id: int, session=None):
-        query_review = validate_reviewer_permission(
-            query_review_id, reviewer_id, session
+    def _validate_review(
+        self, query_execution_id: int, reviewer_id: int, session=None
+    ) -> Tuple[QueryExecution, QueryReview]:
+        """
+        Validate review request and permissions.
+
+        Performs all necessary validation checks:
+        1. Query execution exists
+        2. Review exists and is pending
+        3. Reviewer has permission
+
+        Args:
+            query_execution_id (int): ID of the query execution.
+            reviewer_id (int): ID of the reviewing user.
+            session: Database session.
+
+        Returns:
+            Tuple[QueryExecution, QueryReview]: The query execution and review objects.
+
+        Raises:
+            ValueError: If any validation fails.
+        """
+        query_execution = qe_logic.get_query_execution_by_id(
+            query_execution_id, session=session
+        )
+        api_assert(
+            query_execution is not None,
+            f"QueryExecution with id {query_execution_id} does not exist.",
+            RESOURCE_NOT_FOUND_STATUS_CODE,
+        )
+
+        query_review = query_execution.review
+        api_assert(
+            query_review is not None,
+            "Review not found",
+            RESOURCE_NOT_FOUND_STATUS_CODE,
         )
-        query_execution = query_review.query_execution
 
+        current_status = query_review.get_status()
+        api_assert(
+            current_status == "pending",
```

Comment:
why not just check the `query_execution.status == QueryExecutionStatus.PENDING_REVIEW` directly instead of introducing the literal `"pending"`  string?